lib/opensusebasetest.pm

```diff
@@ -971,7 +971,7 @@ behave as if the env var NOAUTOLOGIN was set.
 sub wait_boot_past_bootloader {
     my ($self, %args) = @_;
     my $textmode     = $args{textmode};
-    my $ready_time   = $args{ready_time} // ((check_var('VIRSH_VMM_FAMILY', 'hyperv') || check_var('BACKEND', 'ipmi')) ? 500 : 300);
+    my $ready_time   = $args{ready_time} // 500;
```

Comment:
This change might be ok but better do not hide it in a commit which does not really cover this. With this change any further performance regression in bootup time would not be detected unless the even longer time of 500 would be exceeded.

Reply:
optionally we could add ` || get_var(RAIDLEVEL)` to narrow a bit more, but once seen in 64bits it might be ok to simplify like that.

Reply:
You have a point Oliver, I have separated the commits. 

I don't know if we would gain much by complicating the code even more and adding "|| get_var('RAIDLEVEL')" instead of simplifying the timeout. Since we had a situation for 64bit, where emergency shell needed more than 300 seconds to appear, I guess it makes sense to use the same output for all archs. I guess your reasoning is not to waste in case SUT is stuck, but does that happen so frequently?

Reply:
The code change is fine as you provided it. I just prefer this to be a bit more explicit with the commit message explaining the reason. Unfortunately I must say that https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11395/commits/52c12ecf04e171faf548a2c307295fa26036312b doesn't really cover it yet because it only states the obvious, what changed in code. Try to put yourself into the shoes of future-Sofia that is trying to find out why the heck someone changed 300 to 500 because we missed a critical performance regression :)

So, the commit message could have a bit more details, e.g. what issue is fixed, why this needs to change, to which versions of which product it applies, link to a bug or a fate entry, the choices you made, etc. Also see https://commit.style/ or http://chris.beams.io/posts/git-commit/ as a helpful guide how to write good commit messages. It does not have to be more than the subject if that describes the reasoning of the change. Often it is better to add least add one sentence in the git commit message body. And additionally a link to the according progress ticket can help a lot to provide context.

Reply:
Done

Reply:
thank you